plugins/serializers/splunkmetric/splunkmetric.go

```diff
@@ -9,12 +9,16 @@ import (
 )
 
 type serializer struct {
-	HecRouting bool
+	HecRouting              bool
+	SplunkmetricMultiMetric bool
 }
 
-func NewSerializer(splunkmetric_hec_routing bool) (*serializer, error) {
+// NewSerializer Setup our new serializer
+func NewSerializer(splunkmetric_hec_routing bool, splunkmetric_multimetric bool) (*serializer, error) {
```

Comment:
We should probably switch to passing in enum types, similar to what was done in the `influx` serializer with UintSupport.  This is just much more readable at the call point.

Reply:
I don't understand what your ask here is. Happy to make changes with a little more direction.

Reply:
This isn't super important, so we can do it later.  What I've found is that having multiple boolean parameters to a function is somewhat hard to read.  Compare these calls of the functions:
```
splunkmetric.NewSerializer(true, false)
splunkmetric.NewSerializer(HecRouting, NoMultiMetric)
```

Another advantage of this is that it is harder to mix up the argument order without getting a compile error.

Here is an example of defining a boolean type in the influx serializer:
https://github.com/influxdata/telegraf/blob/7ff6ec19630d4c0e9da61ba3e6a048797b37bf99/plugins/serializers/influx/influx.go#L17-L22

Again though, this isn't urgent, we can add it in a future change.